src/main/java/com/actiontech/dble/server/handler/BeginHandler.java

```diff
@@ -20,6 +20,7 @@ public static void handle(String stmt, ShardingService service) {
             service.beginInTx(stmt);
         } else {
             service.setTxStart(true);
+            service.setAutocommit(false);
```

Comment:
txStart and autocommit have different meanings

Reply:
+1